### DISCUSSION TITLE
What is the battery sequence of the iCub robot battery pack?

### DISCUSSION THREAD

**Discussion by xilihutu888** [robotology/community#479] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/20624124/105631594-2d185800-5e8a-11eb-992a-b4090c3a5657.png)
Request instructions for a robot battery pack arrangement.Please,

**Comment by pattacini** [comment, marked as answer]
Hi @nan2854 

There's already an ongoing thread in https://github.com/icub-tech-iit/electronics-boards-public/issues/1#issuecomment-766613812.

We tend to avoid creating multiple threads for the same subject; therefore, please stick to the discussion already commenced in https://github.com/icub-tech-iit/electronics-boards-public/issues/1.

Thanks!
